cmd/bench/main.go

```
@@ -364,14 +507,61 @@ func executeWorkload(parent context.Context, target benchmarkTarget, cfg workloa
+					return
+				}
+				if err := pub.Publish(prefillCtx, body); err != nil {
+					if errors.Is(err, context.Canceled) || errors.Is(err, context.DeadlineExceeded) {
+						return
+					}
+					recordErr(err)
+					return
+				}
+				produced.Add(1)
+			}
+			pub.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `pub.Close` is not checked

(errcheck)